spells/increase-login-time-player.js

```diff
@@ -0,0 +1,28 @@
+function increaseLoginTimePlayer(context, payload) {
+  const duration = 60 * 1000;
+  // Change to a correct value later on, this is just a set value to make a change in state
+  const LOGIN_MULTIPLIER = 1.5;
+  const DEFAULT_LOGIN_TIME = 1;
+  const targetedPlayerId = payload.playerId;
+  const gameState = context.getGameState();
+
+  gameState.players[targetedPlayerId].properties.loginTimeMultiplier *= LOGIN_MULTIPLIER;
+
+  const newGameState = { ...gameState };
+
+  context.updateGameState(newGameState);
+  context.sendTo(targetedPlayerId, 'player:slowed', { duration });
+
+  setTimeout(() => {
+    gameState.players[targetedPlayerId].properties.loginTimeMultiplier = DEFAULT_LOGIN_TIME;
```

Comment:
Oh, now we can actually use the attribute `defaults`. So all of our entities (player, team, station) has both `defaults` and `properties` (the latter being the one we actively change). You should be able to write this as:

``` js
gameState.players[targetedPlayerId].properties.loginTimeMultiplier = gameState.players[targetedPlayerId].default.loginTimeMultiplier;
```

Which quite long..., so on line 5 where you store your default login time, instead of manually setting it to 1, fetch it from the player `defaults` property.